### ISSUE TITLE
swdeploy service using 100% CPU on all GPU nodes

### ISSUE THREAD

**Issue by rh-codebase** [ovro-lwa/lwa-issues#111]
service restarted by jack. No other info available. Maybe related to when MCS was rebooted. Check grafana to see if gpu system load can be correlated with MCS reboot.

**Comment by rh-codebase** [comment]
![mcsGrafana](https://user-images.githubusercontent.com/54164837/165836400-c5ac762e-264f-4f73-bf25-3de65936ae50.png)
![gpu01Grafana](https://user-images.githubusercontent.com/54164837/165836486-14810e2a-4c56-48e3-9c36-940e48736cb2.png)
There does look to be correlation between GPU CPU load and lack of network traffic. The MCS was restarted on the right side of MCS graph. The lack of traffic could be the result of Etcd being down although during the large time MCS was down, there looks to be a gradual increase in CPU load. Kind of strange. The linux host's Prometheus client seem to have some buffering as data shows up even when MCS was down. This should be easily tested by shutting down Etcd next maintenance day.

**Comment by jack-h** [comment]
This is pinned at 100% CPU use again. I won't restart the service unless instructed in case there is some useful debugging to be done with it still live

**Comment by caseyjlaw** [comment]
I'm still seeing 100% CPU for this service on the gpu servers.

**Comment by rh-codebase** [comment]
yep. restarted on gpu01-08. This process sure doesn't like having etcd pulled out from beneath it. Will look into it....after dsa2k PDR. :)
